app/models/solr_document.rb

```diff
@@ -30,6 +30,26 @@ def alpha_creator
     self['alpha_creator_tesim']
   end
 
+  def creator
+    self['alpha_creator_tesim'] ? self['alpha_creator_tesim'] : []
```

Comment:
@MPLSFedResearchTZ - this might be slightly more idiomatic in ruby (and therefore slightly easier for other folks to read)
```
self['alpha_creator_tesim'] || []
```

Using the ` a || b` structure says return `b` if `a` is false or nil.